firmware/boot/bootloader.c

```diff
@@ -161,6 +162,9 @@ void bootloader_init(void)
     // other MCUs.
     bootloader_boardSpecific_init();
 
+    hw_can_init(&can);
+    io_canQueue_init();
+
```

Comment:
If you start the HW CAN here then I think you should stop and deinit it before jumping to the app.